extensions/git/src/encoding.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
you should do a find\replace for "MIT License" and refer to "Source EULA".  Except in 3rd-Party notices where we refer to other components licenses, which we should keep as MIT.